pkg/util/validation/validation.go

```diff
@@ -88,7 +88,7 @@ func IsValidLabelValue(value string) []string {
 }
 
 const dns1123LabelFmt string = "[a-z0-9]([-a-z0-9]*[a-z0-9])?"
-const dns1123LabelErrMsg string = "a valid DNS (RFC 1123) label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character"
+const dns1123LabelErrMsg string = "a DNS label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character"
```

Comment:
the problem is that we have at least 2 errors that say "a DNS label".  How about "a DNS-1123" and "DNS-1035" ?